.github/actions/build-electron/action.yml

```diff
@@ -59,17 +59,12 @@ runs:
         node electron/script/check-symlinks.js
     - name: Strip Electron Binaries ${{ inputs.step-suffix }}
       shell: bash
-      if: ${{ inputs.strip-binaries == 'true' && inputs.target-platform == 'linux' }}
+      if: ${{ inputs.strip-binaries == 'true' }}
```

Comment:
based on what i'm seeing, `strip-binaries` will only ever be `true` on Linux, so we can probably remove this second condition